lib/ask/runtime/panel_survey.ex

```diff
@@ -44,6 +44,15 @@ defmodule Ask.Runtime.PanelSurvey do
     |> Survey.update_questionnaires(questionnaire_ids)
   end
 
+  def new_occurrence_name() do
+    now_yyyy_mm_dd()
+  end
+
+  def new_panel_survey_name(nil = _survey_name), do: "Panel Survey #{now_yyyy_mm_dd()}"
+  def new_panel_survey_name(survey_name), do: survey_name
+
+  defp now_yyyy_mm_dd(), do: SystemTime.time.now |> Timex.format!("{YYYY}-{0M}-{D}")
```

Comment:
To be honest, I'm not sure where the dividing line between `runtime/panel_survey` and `model/panel_survey` is. So if it's clear to you that they belong to the model, let's put them there!

Reply:
🤔 Let's talk about this in person too, please. I'm not sure about moving all these functions to the model. I think it could work with `new_occurrence_name` and `new_panel_survey_name`, but I'm not sure about moving more than these.

Reply:
As far as I can see, the name `new_ocurrence_changeset` is confusing. It appears to simply generate a basic changeset, but I think it handles part of the panel survey runtime logic. I'm not sure about how to refactor/rename it, but I'm pretty sure it's part of the Runtime.